### ISSUE TITLE
With emu r775 black screen on PowerVu SIS Channels

### ISSUE THREAD

**Issue by periccapio** [oscam-emu/oscam-emu#290]
With emu r775 - Oscam r11438 black screen on PowerVu SIS Channels (Astra 4A 12418 H 27500).
Note that emu r774 - Oscam r11436 works OK

**Comment by oscam-emu** [comment]
Did you change your reader to not ignore cw checksums for powervu?

See here: https://github.com/oscam-emu/oscam-patched/wiki/PowerVU-config#november-2018-update---attention

**Comment by periccapio** [comment]
Nope,sorry.I apologize for it.But AFN channels work without disablecrccws_only_for = 0E00:000000 directive

**Comment by oscam-emu** [comment]
No problem.

AFN use the CSA algorithm for stream encryption, so they actually use every forth byte as checksum.
Other PowerVu channels use DES, so we must tell OSCam that there are no checksum bytes in the CW.
